features/recognize.js

```diff
@@ -1,11 +1,11 @@
-const { emoji, maximum, minimumMessageLength, usersExemptFromMaximum, reactionEmoji } = require('../config')
```

Comment:
This'll break a few other things, right? The `emoji` variable isn't just used for removing :fistbump: from messages, but also for identifying messages containing a fistbump in the first place, etc.

Reply:
I added the emoji variable back to the config but renamed it to `recognizeEmoji` to clarify it's purpose. You're right about it breaking things...not sure why I didn't see that when I was testing.